packages/fxa-payments-server/src/components/PlanDetails/index.test.tsx

```diff
@@ -162,6 +163,70 @@ describe('PlanDetails', () => {
     expect(queryByTestId('plan-details-component')).not.toHaveAttribute('role');
   });
 
+  it('does not show the coupon success message when there is no coupon used', () => {
+    const subject = () => {
+      return render(
+        <PlanDetails
+          {...{
+            profile: userProfile,
+            showExpandButton: false,
+            isMobile: false,
+            selectedPlan,
+          }}
+        />
+      );
+    };
+
+    const { queryByTestId } = subject();
+    expect(queryByTestId('coupon-success')).not.toBeInTheDocument();
+  });
+
+  describe('Valid Coupon Used', () => {
+    // it('updates price', () => {
+    //   updateConfig({
+    //     featureFlags: {
+    //       subscriptionCoupons: true,
+    //     },
+    //   });
+
+    //   const subject = () => {
+    //     return render(
+    //         <PlanDetails
+    //           {...{
+    //             profile: userProfile,
+    //             showExpandButton: false,
+    //             isMobile: false,
+    //             selectedPlan,
+    //             coupon: { amount: 200}
+    //           }}
+    //         />
+    //     );
+    //   };
+
+    //   const { queryByTestId} = subject();
+    //   expect(queryByTestId('total-price')).toContain('7.35');
+    // });
+
+    it('displays a success message', () => {
```

Comment:
Could we update this test to include checking that the correct totals and discount amounts are shown?